keymaps/intellij-idea-keymap.cson

```diff
@@ -82,3 +99,89 @@
 
   # Find action
   'cmd-shift-a': 'command-palette:toggle'
+
+################################################################################
+# Windows & Linux                                                              #
+################################################################################
+'.platform-win32 atom-workspace atom-text-editor:not([mini]), .platform-linux atom-workspace atom-text-editor:not([mini])':
+
+  ###########
+  # Editing #
+  ###########
+
+  # Duplicate current line or selected block
+  'ctrl-d': 'editor:duplicate-lines'
+
+  # Select successively increasing code blocks
+  'alt-up': 'editor:select-to-next-subword-boundary'
+
+  # Decrease current selection to previous state
+  'alt-down': 'editor:select-to-previous-subword-boundary'
+
+  # Delete line at caret
+  'shift-delete': 'editor:delete-line'
+  'ctrl-y': 'editor:delete-line'
+
+  # Smart line join
+  'ctrl-shift-j': 'editor:join-lines'
+
+  # Move line up
+  'ctrl-shift-up': 'editor:move-line-up'
+
+  # Move line down
+  'ctrl-shift-down': 'editor:move-line-down'
+
+  ##################
+  # Search/Replace #
+  ##################
+
+  # Replace
+  'ctrl-r': 'find-and-replace:show'
+
+  # Find in path
+  'ctrl-shift-f': 'project-find:show'
+
+  # Replace in path
+  'ctrl-shift-r': 'project-find:show'
+
+  ##############
+  # Navigation #
+  ##############
+
+  # Go to class
+  'ctrl-n': 'fuzzy-finder:toggle-file-finder'
+
+  # Go to file
+  'ctrl-shift-n': 'fuzzy-finder:toggle-file-finder'
+
+  # Go to next editor tab
+  'alt-right': 'pane:show-next-item'
```

Comment:
Could you maybe add this to the MAC keymap as `ctrl-right` such that it is consistent?